installation/helm.md

```diff
@@ -19,37 +19,36 @@ Add Edge Delta helm repository
 helm repo add edgedelta https://edgedelta.github.io/charts
 ```
 
-Run Helm installation
+Run Helm installation and create "edgedelta" namespace
 
 ```text
-helm install edgedelta edgedelta/edgedelta --set apiKey=<API_KEY>
+helm install edgedelta edgedelta/edgedelta --set apiKey=<API-KEY> -n edgedelta --create-namespace
 ```
 
 Output
 
 ```text
 NAME: edgedelta
-LAST DEPLOYED: Tue Jul 7 22:23:37 2020
-NAMESPACE: default
+LAST DEPLOYED: Fri Jul 17 17:49:42 2020
+NAMESPACE: edgedelta
 STATUS: deployed
 REVISION: 1
 TEST SUITE: None
 NOTES:
 1. Visit https://admin.edgedelta.com
-2. Find the configuration with <API_KEY> to check reporting status
+2. Find the configuration with <API-KEY> to check reporting status
```

Comment:
what do we mean exactly by reporting status? It's a little confusing me. At this point they already have API KEY, right? which means they already used admin UI to create a config and copied its id. I don't understand why they would go to admin UI and find the config after deploying with helm. 
Once they deploy with helm, next step would be to visit admin UI and check the active nodes.

Reply:
I mean to see 1. visit https://admin.edgedelta.com 2 check agents reporting status for this config.

Reply:
"check if agents are active" sounds good